view/frontend/web/js/view/cart/product-protection-simple-offer.js

```diff
@@ -43,13 +43,16 @@ define(['cartUtils', 'extendSdk', 'ExtendMagento'], function (cartUtils, Extend,
   }
 
   return function (config) {
+    const sku = config[0].selectedProductSku
     const activeProductData = {
       referenceId: config[0].selectedProductSku,
       price: config[0].selectedProductPrice * 100,
       category: config[0].productCategory,
       onAddToCart: handleAddToCartClick,
     }
     Extend.config({ storeId: config[0].extendStoreUuid, environment: config[0].activeEnvironment })
+    if (sku === 'extend-protection-plan') return
```

Comment:
Should this be before we call `Extend.config()`?

Reply:
Changed the location